crates/pallet-fixed-validators-set/src/lib.rs

```diff
@@ -0,0 +1,96 @@
+//! A substrate pallet for providing a fixed validators set.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::traits::StorageVersion;
+
+pub use self::pallet::*;
+pub use self::weights::*;
+
+#[cfg(feature = "runtime-benchmarks")]
+pub mod benchmarking;
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
+mod weights;
+
+/// The current storage version.
+const STORAGE_VERSION: StorageVersion = StorageVersion::new(0);
+
+// We have to temporarily allow some clippy lints. Later on we'll send patches to substrate to
+// fix them at their end.
+#[allow(clippy::missing_docs_in_private_items)]
+#[frame_support::pallet]
+pub mod pallet {
+    use frame_support::{pallet_prelude::*, storage::in_storage_layer};
+    use frame_system::pallet_prelude::*;
+
+    use super::*;
+
+    /// Configure the pallet by specifying the parameters and types on which it depends.
+    #[pallet::config]
+    pub trait Config: frame_system::Config {
+        /// Overarching event type.
+        type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+
+        /// The maximum number of validators to provide.
+        type MaxValidators: Get<u32>;
```

Comment:
Should the number be dependent on `MaxBioauthValidators` and `MaxBootnodeValidators`  of `pallet-humanode-session` config somehow? 

Just to clarify, should we take care on it or not.

Reply:
I guess, I got your point. It's and independent pallet where all conflicts should be resolved on runtime level.